src/Dapr.AspNetCore/Dapr.AspNetCore.csproj

```diff
@@ -17,6 +17,10 @@
     <ProjectReference Include="..\Dapr.Client\Dapr.Client.csproj" />
   </ItemGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+  <DebugType>portable</DebugType>
```

Comment:
why portbale?

Reply:
allows for the windows debugger to load symbols